modules/core/packer/roles/nomad/tasks/main.yml

```diff
@@ -11,11 +11,18 @@
     version: "{{ nomad_module_version }}"
 - name: Install Nomad
   shell: "/tmp/terraform-aws-nomad/modules/install-nomad/install-nomad --version {{ nomad_version }} --user {{ nomad_user }}"
-- name: "Copy Nomad Configuration"
+- name: "Copy Nomad configuration"
   copy:
     src: "{{ role_path }}/files/config/"
     dest: "/opt/nomad/config"
   become: yes
+- name: "Copy Nomad syslog sub-configuration"
```

Comment:
Maybe just call this `template` configuration?

Reply:
Okay used `with_items` to go with the template configuration idea, did the same for `Consul` side.